e2e/images/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:16.04
```

Comment:
I think alpine didn't have rbd kernel, we will issues open to slim down the test image.